core/src/main/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisher.java

```diff
@@ -160,6 +160,25 @@ public void setAdditionalExceptionMappings(Properties additionalExceptionMapping
 		}
```

Comment:
As a point of hygiene, will you please [update the copyright header for both classes](https://github.com/spring-projects/spring-security/blob/master/CONTRIBUTING.md#update-apache-license-header-to-modified-files-as-necessary)?